GenAI.py

```diff
@@ -18,8 +23,9 @@ def __init__(self):
     def to_markdown(self, text):
         text = text.replace('•', '  *')
         return Markdown(textwrap.indent(text, '> ', predicate=lambda _: True))
-    
+
     def generate_response(self, user_input):
         response = self.model.generate_content(self.CONST_COMMAND + self.CONST_OS + user_input)
-    
-        return response.text
\ No newline at end of file
+        if not response.parts:
```

Comment:
no error is thrown for invalid input, check for it

Reply:
I have pushed the fixes. Seems like the response of AI models is inconsistent. I have added a check for handling this which should be able to handle most of those cases.